src/control/server/server_utils.go

```diff
@@ -218,9 +209,14 @@ func prepBdevStorage(srv *server, iommuEnabled bool) error {
 		}
 	}
 
+	// When requesting to prepare NVMe drives during service start-up, use all addresses
+	// specified in engine config BdevList parameters as the PCIAllowList and the server
+	// config BdevExclude parameter as the PCIBlockList. BdevInclude config parameter has
+	// been removed.
```

Comment:
"BdevInclude config parameter has been removed" - I don't think it should continue to be referenced in comments.

Reply:
done